test/Main.hs

```diff
@@ -11,23 +11,69 @@ import Spec as Spec (tests)
 
 main :: IO ()
 main = do
+  mwasmPathMVP <- lookupEnv "WASM_SPEC_TESTS_MVP"
+  testDirMVP <- case mwasmPathMVP of
+      Nothing -> error "Please define WASM_SPEC_TESTS_MVP to point to .../WebAssembly/spec/test/core"
+      Just path -> pure path
   mwasmPath <- lookupEnv "WASM_SPEC_TESTS"
   testDir <- case mwasmPath of
-      Nothing -> error "Please define WASM_SPEC_TESTS to point to .../WebAssebly/spec/test/core"
-      Just path -> return path
+      Nothing -> error "Please define WASM_SPEC_TESTS to point to .../WebAssembly/spec/test/core"
+      Just path -> pure path
+  putStrLn $ "Using wasm MVP spec test directory: " ++ testDirMVP
   putStrLn $ "Using wasm spec test directory: " ++ testDir
```

Comment:
Why do we need two versions of the test suite? Can we not simply run the latest, possibly excluding new tests that we don't support?

Reply:
Because I had to disable too many test that won't run it the new suite, and I was concerned about regressing on those.